src/pages/gen2/start/account-setup/index.mdx

```diff
@@ -9,105 +9,206 @@ export function getStaticProps(context) {
       meta
     }
   };
-
 }
 
-This guide will walk you through how to set up your AWS account by creating a local profile and by completing a one-time bootstrapping process to perform CI/CD deployments for your Amplify (Gen 2) applications.
+<Callout info>
 
-## Configure local machine
+**Note**: If you already have an AWS account and profile configured locally, you can skip this guide.
 
-You will need to configure your local machine to connect to your AWS account before you can set up the [Amplify Sandbox environment](/gen2/deploy-and-host/sandbox-environments/setup/).
+</Callout>
 
-### Setup a local profile
+{/* "...configure an AWS profile locally for use with Amplify" -> `amplify configure profile` */}
+This guide will walk you through how to set up your AWS account and configure an AWS profile locally for use with Amplify. Before getting started it is important to know there are a few options for authenticating with AWS. Using [AWS Identity and Access Management (IAM)](https://aws.amazon.com/iam/), you can configure either:
 
-<Callout info>
+- permanent credentials with an [IAM user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html) through the use of access keys
+- temporary credentials with an [IAM role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html)
```

Comment:
since we don't mention anything else in this guide about assuming a role directly, maybe just remove this line?

Reply:
ah yes I removed the section for it. removed this item